src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -69,5 +69,43 @@ public static IServiceCollection Configure<TOptions>(this IServiceCollection ser
 
         public static IServiceCollection ConfigureAll<TOptions>(this IServiceCollection services, Action<TOptions> configureOptions) where TOptions : class
             => services.Configure(name: null, configureOptions: configureOptions);
+
+        /// <summary>
+        /// Registers an action used to Initialize a particular type of options.
```

Comment:
Don't need to capitalize the `I` of `Initialize` (it's just a word here, not a method name).